ui/lib/core/addon/utils/uuid.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { v4 as uuidv4 } from 'uuid';
+
+/**
+ * Use this instead of uuidv4() so that generated UUIDs can be stubbed in tests.
```

Comment:
I copied this idea from the timestamp util:
https://github.com/hashicorp/vault/blob/0b12cdcfd17278d7cca25ec37ffacd96b8d5c6c7/ui/lib/core/addon/utils/timestamp.js#L9-L20

Reply:
I'm not sure if this is necessary (disclaimer: going on memory, didn't test). In a test if you import everything from `uuid` then you can stub `v4`:
```js
import sinon from 'sinon';
import * as uuid from 'uuid';

const v4Stub = sinon.stub(uuid, 'v4');
```

Reply:
Ohh yes, I'll try that! I knew this was probably over-engineering - thank you! 🙏

Reply:
Circlin' back...apparently `uuid` is not-stubbable: 

```
VAULT-34548/build-oidc-jwt-component